pkgs/applications/networking/instant-messengers/session-desktop/default.nix

```
@@ -33,7 +33,7 @@ stdenvNoCC.mkDerivation {
       name = "Session";
       desktopName = "Session";
       comment = "Onion routing based messenger";
-      exec = "${appimage}/bin/session-desktop-${version}";
+      exec = "session-desktop";
```

How about keeping the same structure as before? Something like this?

```suggestion
      exec = "${appimage}/bin/session-desktop";
```

---

I requested this change above

---

Oooh sorry !!! Ignore me then :)